docs/modules/ROOT/pages/how-to-guides/proc_managing-compliance-with-the-enterprise-contract.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: PROCEDURE
+
+[id="managing-compliance-with-the-enterprise-contract_{context}"]
+= Managing compliance with the Enterprise Contract
+
+The Enterprise Contract (EC) is a set of tools to help keep your software supply chain secure and ensures that the container images comply with your organization's policies. It does this by verifying the security and provenance of builds created through {ProductName}.
+
+{ProductName}'s build process utilizes the link:https://tekton.dev/docs/chains/[Tekton] Chains to generate a signed link:https://in-toto.io/in-toto/[in-toto] attestation of the build pipeline. The Enterprise Contract then uses this attestation to verify the build's integrity and compliance with a set of policies. These policies include best practices and any organization-specific requirements.
+
+.Prerequisites
+
+* You have created an application. 
+* You have an upgraded build pipeline.
+* You have the link to the image bundle, which contains the Tekton pipeline for your integration test. 
+
+.Procedure
+
+. Open an existing application and go to the *Integration tests* tab.
+. Select *Add integration test*.
+. In the *Integration test name* field, enter a name of your choosing.
+. In the *Image bundle* field, enter the path to the relevant container image. It must contain a Tekton pipeline that executes your test.
+.. To use the Red Hat Enterprise Contract integration test, enter *quay.io/redhat-appstudio-tekton-catalog/pipeline-enterprise-contract:devel.*
```

Comment:
Is there another guide for running an arbitrary integration test? I think like 21 belongs in that guide, but line 22 is correct for an EC guide.